MANIFEST.in

```diff
@@ -1,2 +1,3 @@
 include LICENSE.txt *.rst
 recursive-include docs *.rst *.py
+recursive-include tests *.py
```

Comment:
We can avoid having to manually update this manifest by using [setuptools-scm](https://pypi.org/project/setuptools-scm/), but that can be a follow-up PR.

Reply:
I'm not sure I understand: Why are both docs and tests included for packaging in the first place? Shouldn't we simply remove those two lines?

Reply:
This is for the source distribution made by `sdist`, not for the built distributions made by `bdist` or `wheel`: the latter won't include these files.

Reply:
Do you think it makes sense (as in "is important") to include those files with the source distribution? I would see them as part of the project / development setup only. The times that we would download source tarballs, compile their content and would want to find tests and docs included, aren't they long over?